docs/getting_started/guided/guided.md

```diff
@@ -1,3 +1,9 @@
+!!! Note
+    Difference between Zero-shot and Guided BERTopic: 
+    Guided BERTopic is similar - yet not equivalent - to [Zeros-shot Topic Modeling](https://maartengr.github.io/BERTopic/getting_started/zeroshot/zeroshot.html). 
+    Use Guided BERTopic to boost certain keyword's importance. Use [Zeros-shot Topic Modeling](https://maartengr.github.io/BERTopic/getting_started/zeroshot/zeroshot.html) to try to categorize documents into predefined topics ("zero-shot topics") before the clustering the remaining, unclassified documents, using the default unsupervised BERTopic topic exploration algorithm.
```

Comment:
```suggestion
    Use Guided BERTopic to boost the importance of certain keywords. Use [Zeros-shot Topic Modeling](https://maartengr.github.io/BERTopic/getting_started/zeroshot/zeroshot.html) to try to categorize documents into predefined topics ("zero-shot topics") before clustering the remaining unclassified documents using the main algorithm of BERTopic.
```

Reply:
What I'm missing in the guided approach is that it is not purely focused on increasing a given keyword importance. Due to it's [procedure](https://maartengr.github.io/BERTopic/getting_started/guided/guided.html), it also steers certain documents towards certain clusters. Moreover, guided topic modeling is not primarily meant for increasing a keyword importance. It instead tries to steer documents towards given clusters. The focus here are seed topics, and not seed keywords.

Reply:
Thank you for the changes and for the suggestion regarding the explanation of the guided approach. I'll have a look at this when time allows.